cypress/integration/home_spec.js

```diff
@@ -0,0 +1,19 @@
+describe("Home", () => {
+  it("Check fields", () => {
+    cy.visit("/");
+    cy.get("div.banner").should("be.visible");
```

Comment:
Maybe we should be a little less specific with the selectors?

Something as simple as this would do:

```javascript
cy.visit("/").contains('A place to share your knowledge')
```

I'm thinking about making this test suite useful for other implementations.

@geromegrignon do you have any thoughts on that?

Reply:
About selectors, I know @ChrisZie cares about using proper selectors (you can see her related blog post [here](https://dev.to/chriszie/data-qa-attribute-a-better-way-to-select-elements-for-ui-test-automation-48lm)).

Rather than spending time adding selectors on the current demo, I prefer to focus on the new versions of the specs. I would like to delay the addition of proper selectors as we don't have that much traffic going on right now on implementation submission to prioritize such task.